### PR TITLE
docs(resources): add awesome dart frog

### PR DESCRIPTION
**Pull request by scarletteliza**
## Status

**READY**

## Description

Add link to Awesome Dart Frog repo & remove others (since they're linked elsewhere)

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by renancaraujo — APPROVED** [review]

**Review by wolfenrain — APPROVED** [review]

File: docs/docs/basics/resources.md [hunk 1]
```diff
@@ -5,18 +5,12 @@ title: 📚 Additional Resources
 
 # Additional Resources 📚
 
-- [Examples][examples_link]
-- [Roadmap][roadmap_link]
-- [Blog post][blog_link]
-- [Livestream demo][livestream_link]
+- [Awesome Dart Frog repo]: Highlights awesome Dart Frog resources — articles, videos, open source projects, and more!
```
Comment (felangel) [inline docs/docs/basics/resources.md:8]: Looks like we aren’t referencing the awesome_df link here

**Review by felangel — CHANGES_REQUESTED** [review]
See comment

File: docs/docs/basics/resources.md [hunk 2]
```diff
@@ -5,18 +5,12 @@ title: 📚 Additional Resources
...
 Fun fact: Did you know the [dart2js][dart2js_compiler_link] compiler [used to be called frog][dart2js_frog_pr_link]?
 :::
 
-[blog_link]: https://verygood.ventures/blog/dart-frog
 [dart2js_compiler_link]: https://dart.dev/tools/dart2js
 [dart2js_frog_pr_link]: https://github.com/dart-lang/sdk/issues/2194
-[examples_link]: https://github.com/VeryGoodOpenSource/dart_frog/tree/main/examples
-[livestream_link]: https://youtu.be/N7l0b09c6DA
-[roadmap_link]: /docs/roadmap
+[awesome_df]: https://github.com/VeryGoodOpenSource/awesome-dart-frog
```
Comment (felangel) [inline docs/docs/basics/resources.md:16]:
```suggestion
[awesome_dart_frog_link]: https://github.com/VeryGoodOpenSource/awesome-dart-frog
```

File: docs/docs/basics/resources.md [hunk 3]
```diff
@@ -5,18 +5,12 @@ title: 📚 Additional Resources
 
 # Additional Resources 📚
 
-- [Examples][examples_link]
-- [Roadmap][roadmap_link]
-- [Blog post][blog_link]
-- [Livestream demo][livestream_link]
+- [Awesome Dart Frog repo][awesome_df]: Highlights awesome Dart Frog resources — articles, videos, open source projects, and more!
```
Comment (felangel) [inline docs/docs/basics/resources.md:8]:
```suggestion
- [Awesome Dart Frog repo][awesome_dart_frog_link]: Highlights awesome Dart Frog resources — articles, videos, open source projects, and more!
```

**Review by felangel — APPROVED** [review]

**Pull request merged**